### PR TITLE
Add snapshot.info() feature

### PR DESCRIPTION
**Pull request by sac**
Signed-off-by: Sachidananda Urs <sacchi@gmail.com>

### CODE REVIEW

File: glustercli/cli/parsers.py [hunk 1]
```diff
@@ -675,8 +675,20 @@ def parse_snapshot_status(data):
...
 
 def parse_snapshot_info(data):
-    raise NotImplementedError("Snapshot Info")
-
+    xml = etree.fromstring(data)
+    snapinfo = []
+    for snap_el in xml.findall('snapInfo/snapshots/snapshot'):
+        snapdata = {
+            'name': snap_el.find('name').text,
+            'createtime': snap_el.find('createTime').text,
```
Comment (aravindavk) [inline glustercli/cli/parsers.py]: `/s/createtime/create_time/`

File: glustercli/cli/parsers.py [hunk 2]
```diff
@@ -675,8 +675,20 @@ def parse_snapshot_status(data):
...
+            'uuid': snap_el.find('uuid').text,
+            'status': snap_el.find('snapVolume/status').text,
+        }
+        if snap_el.find('snapVolume/originVolume'):
+            snapdata['originvolume'] = \
```
Comment (aravindavk) [inline glustercli/cli/parsers.py]: `s/originvolume/origin_volume/`

Comment (sac) [reply glustercli/cli/parsers.py]: Done

File: glustercli/cli/parsers.py (hunk 1 above)
Comment (sac) [reply glustercli/cli/parsers.py]: Done.

**Review by aravindavk — APPROVED** [review]

**Pull request merged**